docs/MODULES.md

```diff
@@ -144,39 +217,44 @@ Defines the yggdrasil interface.
 Default: ygg0
 Value: `Any interface on the system`
 
+#### IPTunnel - Server
+
+The IPTunnel server acts as a exit node. It will accept connections from other yggdrasil peeres listed in **/etc/yggdrasil.iptunnel.server** and form a tunnnel between them allowing the remote peer to access internet available on this node.
```

Comment:
```suggestion
The IPTunnel server acts as a exit node. It will accept connections from other Yggdrasil peers listed in **/etc/yggdrasil.iptunnel.server** and form a tunnel between them allowing the remote peer to access Internet available on this node.
```